packages/odyssey-storybook/src/components/odyssey-mui/Callout/Callout.stories.tsx

```diff
@@ -69,10 +69,10 @@ const storybookMeta: Meta<CalloutProps> = {
     },
     text: {
       control: "text",
-      description: "The text content of the Callout",
+      description: "The content of the Callout",
       table: {
         type: {
-          summary: "string",
+          summary: "string | ReactNode",
```

Comment:
@benlister-okta `text` is still only a string right? We aren't rendering the list from the `text` prop, it's being passed in as `children`. So, don't think we need to update the type here actually